pkg/project/resource_project_repo.go

```diff
@@ -127,10 +127,16 @@ var updateRepos = func(projectKey string, terraformRepoKeys []RepoKey, m interfa
 	return readRepos(projectKey, m)
 }
 
+var retry5xxRange = func(response *resty.Response, err error) bool {
+	retryableHttpStatusCode := []int{408, 500, 502, 503, 504}
+	return containsInt(retryableHttpStatusCode, response.StatusCode())
+}
+
 var addRepo = func(projectKey string, repoKey RepoKey, m interface{}) error {
 	log.Println("[DEBUG] addRepo")
```

Comment:
This will work but we need to integrate [terraform-plugin-log](https://github.com/hashicorp/terraform-plugin-log) to have actual working logging. I've done it in another PR but I removed it later on since we didn't need it then.